src/onepirate/modules/views/SignUp.js

```diff
@@ -1,22 +1,24 @@
-import withRoot from '../withRoot';
+import withRoot from "../withRoot";
 // --- Post bootstrap -----
-import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import Grid from '@material-ui/core/Grid';
-import Link from '@material-ui/core/Link';
-import { Field, Form, FormSpy } from 'react-final-form';
-import Typography from '../components/Typography';
-import AppFooter from './AppFooter';
-import AppAppBar from './AppAppBar';
-import AppForm from './AppForm';
-import { email, required } from '../form/validation';
-import RFTextField from '../form/RFTextField';
-import FormButton from '../form/FormButton';
-import FormFeedback from '../form/FormFeedback';
-import CheckBox from '../form/CheckBox';
-import Radio from '@material-ui/core/Radio';
```

Comment:
All these are due to my format on save 😅 we can have a eslint/prettier config file to standardise our code styles